src/clarity-angular/popover/dropdown/dropdown.ts

```diff
@@ -56,6 +61,11 @@ export class Dropdown {
     constructor(public elementRef: ElementRef, @SkipSelf() @Optional() public parent: Dropdown,
                 public ifOpenService: IfOpenService) {
         this.anchor = elementRef.nativeElement;
+
+        // Subscribe and broadcast openChanges from the service
+        ifOpenService.openChange.takeUntil(this.ngUnsubscribe).subscribe(change => {
```

Comment:
One thing though wrt to: 
```
 it'd require the additional import (dependency)
```
We gain the Subject import but lose the Subscription import. You are probably right though, it should be unsubscribed the same way as the others.